### PR TITLE
feeluown: fix desktop item

### PR DESCRIPTION
**Pull request by wineee**
https://github.com/berberman/flakes/issues/24

### CODE REVIEW

File: packages/feeluown/default.nix [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
+      name = "FeelUOwn";
+      desktopName = "FeelUOwn";
+      exec = "feeluown --log-to-file";
+      categories = "AudioVideo;Audio;Player;Qt;";
+      terminal = "false";
+      icon =
+        "${feeluown-core}/lib/${python.executable}/site-packages/feeluown/icons/feeluown.png";
+      comment = "FeelUOwn Launcher";
+      startupNotify = "true";
+      # startupWMClass = "FeelUOwn";
```
Comment (wineee) [inline packages/feeluown/default.nix]: Forgot to change, don't merge

**Comment by berberman** [comment]
It seems that there are more things to be migrated against the new version of nixpkgs :)

**Comment by wineee** [comment]
fastocr has also been maintained

**Comment by berberman** [comment]
Thank you!

**Pull request merged**